apps/android/package.json

```diff
@@ -25,28 +25,35 @@
     "@react-native-community/slider": "^3.0.3",
     "@react-native-picker/picker": "^1.16.7",
     "hermes-engine": "~0.5.0",
-    "react": "16.13.1",
-    "react-native": "^0.63.2",
+    "react": "17.0.1",
+    "react-native": "^0.64.3",
     "react-native-svg": "^12.1.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.6.2",
-    "@babel/runtime": "^7.6.2",
+    "@babel/core": "^7.8.0",
+    "@babel/runtime": "^7.8.0",
     "@fluentui-react-native/eslint-config-rules": "^0.1.1",
     "@rnx-kit/cli": "^0.5.27",
     "@rnx-kit/metro-config": "^1.2.3",
     "@uifabricshared/build-native": "^0.1.1",
-    "metro-react-native-babel-preset": "^0.59.0",
+    "metro-react-native-babel-preset": "^0.66.2",
     "react-native-svg-transformer": "^0.14.3",
     "react-native-test-app": "^0.7.0",
```

Comment:
The failing iOS and Android tests may be because we need to bump `react-native-test-app`. The first version that officially supports RN-macOS 0.64 for example, is 0.9.12.

Reply:
I did update RNTA but that isn't enough. Something about @react-native-community/cli, which updated from 4.1.0 to 5.x is causing the breakage.

Reply:
Ah, lemme move my other comment here.

My next guess is that we may need to bump @react-native-community/cli. The iOS PR is running `react-native run-ios`, which I remember failed with RNTA & RN 0.64 until we made some upstream CLI fixes, which showed up in @react-native-community/cli version 5.02 https://github.com/microsoft/react-native-test-app/issues/375

`yarn why` on this branch tells me we are on 5.0.1
``` bash
sanajmi@saads-mac-mini fluentui-react-native % yarn why @react-native-community/cli
yarn why v1.22.15
[1/4] 🤔  Why do we have the module "@react-native-community/cli"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@react-native-community/cli@5.0.1"
info Has been hoisted to "@react-native-community/cli"
info Reasons this module exists
   - "workspace-aggregator-02357f17-40e3-4b55-9492-648fa746f567" depends on it
   - Hoisted from "_project_#@fluentui-react-native#tester-win32#@office-iss#react-native-win32#@react-native-community#cli"
   - Hoisted from "_project_#@fluentui-react-native#tester#react-native-windows#@react-native-community#cli"
   - Hoisted from "_project_#@fluentui-react-native#tester#react-native-macos#@react-native-community#cli"
   - Hoisted from "_project_#@fluentui-react-native#tester-android#react-native#@react-native-community#cli"
info Disk size without dependencies: "1.78MB"
info Disk size with unique dependencies: "22.67MB"
info Disk size with transitive dependencies: "77.04MB"
info Number of shared dependencies: 309
=> Found "@uifabricshared/build-native#@react-native-community/cli@6.1.0"
info This module exists because "_project_#@uifabricshared#build-native" depends on it.
info Disk size without dependencies: "892KB"
info Disk size with unique dependencies: "60.51MB"
info Disk size with transitive dependencies: "127.75MB"
info Number of shared dependencies: 310
✨  Done in 2.63s.
```